src/lib/functions/registry.mjs

```diff
@@ -123,6 +124,32 @@ export class FunctionsRegistry {
   }
 
   async getFunctionForURLPath(urlPath, method) {
+    const defaultURLMatch = urlPath.match(DEFAULT_URL_EXPRESSION)
```

Comment:
I believe this block should live in `NetlifyFunction#matchUrlPath`. That'd make the logic a little easier to reason about, as well.

It'd probably result in more regex matchings needed to be done, but I don't think that's a performance concern at the orders of magnitude that functions have in local dev.

Reply:
okay, scratch that - would make https://github.com/netlify/cli/blob/05c44a6d2c17ef80a860bce90feebe58c1f02198/src/lib/functions/registry.mjs#L137-L147 a lot harder to implement